packages/mds-transaction-api/handlers/get-transactions.ts

```diff
@@ -279,7 +279,7 @@ export const GetTransactionsAsCsvHandler = async (
         amount: transaction.amount / 100, // conversion from pennies to dollars or equivalent
         currency_iso: process.env.TRANSACTION_CURRENCY || 'USD', // TODO add support for multiple currencies in mds-transaction-service
         provider_name: hasOwnProperty(providerMap, transaction.provider_id)
-          ? providerMap[transaction.provider_id]
+          ? providerMap[transaction.provider_id]?.provider_name
```

Comment:
Why the need for the `hasOwnProperty`? Can't this simply be:
```ts
provider_name: providerMap[transaction.provider_id]?.provider_name ?? 'unknown'
```

Reply:
Probably, yeah